app/src/services/conditional_token.ts

```diff
@@ -178,6 +193,40 @@ class ConditionalTokenService {
 
     return conditionId
   }
+
+  getConditionIdFromLogs = async (conditionId: string): Promise<ConditionLog> => {
```

Comment:
This function is not necessary, but I think it does not affect if we keep it, and maybe could be usefull

Reply:
We either keep this function and remove `getQuestionId`, or we remove this one. We can't keep both of them, they are almost equal.

Reply:
Remove it